src/tools/genpolicy/README.md

```diff
@@ -0,0 +1,28 @@
+# Agent Policy generation tool
+
+The Kata Containers policy generation tool (`genpolicy`): 
+
+1. Reads user's Kubernetes YAML file.
+
+1. Infers user's intentions based on the contents of that file.
+
+1. Generates a Kata Containers Agent (`kata-agent`) policy file
```

Comment:
I'd be interested in understanding/communicating some of the rules that genpolicy used based on input patterns, or at least some examples as the policy file that is output is pretty large (the basic pod yaml example you give it decodes to >1600 lines long) so is tricky to understand e.g. things like the container image name will be extracted and added to the list of images that are allowed in the CreateContainerRequest, or something like that?

Maybe explaining the genpolicy-settings.json file, which looks like it does some templating? - what it is and what is does would help with this?

Reply:
I've just added many policy details into the new genpolicy-auto-generated-policy-details.md, and advanced command line usage in genpolicy-advanced-command-line-parameters.md.